app/views/components/_reorderable_list.html.erb

```diff
@@ -0,0 +1,47 @@
+<%
+  items ||= []
+  input_name ||= "ordering"
+  data_attributes ||= {}
+  data_attributes[:module] = "reorderable-list"
+%>
+
+<%= tag.ol class: "app-c-reorderable-list", data: data_attributes do %>
+  <% items.each_with_index do |item, index| %>
+    <%= tag.li class: "app-c-reorderable-list__item" do %>
+      <%= tag.div class: "app-c-reorderable-list__wrapper" do %>
+        <%= tag.div class: "app-c-reorderable-list__content" do %>
+          <%= tag.p item[:title], class: "app-c-reorderable-list__title" %>
+          <%= tag.p(item[:description], class: "app-c-reorderable-list__description") if item[:description].present? %>
+        <% end %>
+        <%= tag.div class: "app-c-reorderable-list__actions" do %>
+          <% label_text = capture do %>
+            Position<span class='govuk-visually-hidden'> for <%= item[:title] %></span>
+          <% end %>
+          <%= render "govuk_publishing_components/components/input", {
+            label: {
+              text: sanitize(label_text)
+            },
+            name: "#{input_name}[#{item[:id]}]",
+            type: "number",
+            value: index + 1,
+            width: 2
+          } %>
+          <%= render "govuk_publishing_components/components/button", {
+            text: "Up",
+            type: "button",
+            aria_label: "Move #{item[:title]} up",
+            classes: "js-reorderable-list-up",
+            secondary_quiet: true
```

Comment:
I recall commenting on this last time but I can't find my last comment so I'm assuming I messed it up - but apologies if not and this is deja vu.

I remember us talking about using secondary buttons here rather than the quiet buttons. Is that something we can try out here?

Reply:
Did we want to do this in this iteration, I was under the assumption that we'd keep it similar to the current implementation now then try out the secondary buttons after?

Reply:
I guess it already has a change in this with the green button going to grey so it'd probably be less jarring for it not to change twice - but I don't have qualms with waiting. 

Perhaps quickly change it to the other buttons, take a screenshot and share it in our channel/our stand-up to see if anyone objects to the change - if nothing occurs just roll with them if there is roll with this and have those discussions further after?

Reply:
Changed